slam-libraries/viam-orb-slam3/orbslam_server_v1.h

```diff
@@ -39,6 +41,19 @@ class SLAMServiceImpl final : public SLAMService::Service {
     ::grpc::Status GetMap(ServerContext *context, const GetMapRequest *request,
                           GetMapResponse *response) override;
 
+    /*
+      For a given GetPointCloudMapRequest
+      Returns a GetPointCloudMapResponse containing a sparse
+      slam map as a Binary PCD file
```

Comment:
[nit] remove file as PCD is a valid format to refer to a buffer and having file here might make people expect a saved file

Reply:
done